egs/librispeech/s5/local/nnet3/run_ivector_common.sh

```diff
@@ -8,20 +8,21 @@
 stage=0
 generate_alignments=true # false if doing ctc training
 speed_perturb=true
+cleanup_affix=cleaned
 
 set -e
 . ./cmd.sh
 . ./path.sh
 . ./utils/parse_options.sh
 
 
-train_set=train_960
+train_set=train_960_${cleanup_affix}    # you might set this to e.g. train_960
 if [ "$speed_perturb" == "true" ]; then
   if [ $stage -le 1 ]; then
     #Although the nnet will be trained by high resolution data, we still have to perturbe the normal data to get the alignment
     # _sp stands for speed-perturbed
 
-    for datadir in train_960; do
+    for datadir in ${train_set}; do
       utils/perturb_data_dir_speed.sh 0.9 data/${datadir} data/temp1
```

Comment:
Modify the data perturbation commands. There is new script *perturb_3way.sh which can be used make your script much cleaner.